.github/workflows/new_issue_maintainer.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 jobs:
   maintainer-opened:
     runs-on: ubuntu-latest
-    if: ${{ !github.event.issue.pull_request && contains(fromJSON('["palpatim", "brennanMKE", "diegocstn", "lawmicha", "harsh62", "thisisabhash", "ameter", "royjit", "atierian", "ukhan-amazon", "ruisebas", "phantumcode"]'), github.event.issue.user.login) }}
+    if: ${{ !github.event.issue.pull_request && contains(fromJSON('["palpatim", "brennanMKE", "5d", "lawmicha", "harsh62", "thisisabhash", "ameter", "royjit", "atierian", "ukhan-amazon", "ruisebas", "phantumcode"]'), github.event.issue.user.login) }}
```

Comment:
I'm guessing the answer is not, but is there support for group checking instead of a hardcoded list of users?

Reply:
Who is `5d` 🤔  ? 
And while we're at it, maybe we should add `diwud ` and the data-team as well?

Reply:
Unfortunately, I wasn't able to find a way to automatically pull a list in the condition block of an action. Agreed though, that would be ideal.